webpack.config.js

```diff
@@ -53,15 +56,11 @@ module.exports = {
           MiniCssExtractPlugin.loader,
           {
             loader: 'css-loader',
-            options: {
-              modules: true,
-              localIdentName: '[local]',
-            },
           },
           {
             loader: 'sass-loader',
             options: {
-              data: '@import "paragon-reset";',
+              data: '@import "bootstrap-variables";',
```

Comment:
Is there a reason to do this, rather than just manually import those three imports in bootstrap-variables.scss into index.scss?  Not sure I understand the value of this `data` thing.

Reply:
The value here is that when building the minified scss we need bootstrap functions, variables, and mixins to be set, so this loader adds them. 

We still want index.scss to be a clean partial so that consuming applications can determine how or which bootstrap theme variables to include.